docs/tf_model_binary_input.md

```diff
@@ -24,7 +24,7 @@ rmdir resnet_v2_fp32_savedmodel_NHWC/
 
 Convert the TensorFlow model to Intermediate Representation format using model_optimizer tool:
 ```Bash
-docker run -u $(id -u):$(id -g) -v ${PWD}/resnet_v2/:/resnet openvino/ubuntu20_dev:2022.1 mo --saved_model_dir /resnet/ --output_dir /resnet/models/resnet/1/ --input_shape=[1,224,224,3] --mean_values=[123.68,116.78,103.94] --reverse_input_channels
+docker run -u $(id -u):$(id -g) -v ${PWD}/resnet_v2/:/resnet openvino/ubuntu20_dev:latest mo --saved_model_dir /resnet/ --output_dir /resnet/models/resnet/1/ --input_shape=[1,224,224,3] --mean_values=[123.68,116.78,103.94] --reverse_input_channels
```

Comment:
Why latest?

Reply:
It is either 2022.1.0 or latest. There is no 2022.1 version. 
Should I change it to 2022.1.0 then?

Reply:
Yes, let's stick to the fixed versions so we have full control over the behavior of that command across multiple releases.

Reply:
Done